### PR TITLE
Tk 8 export quota data as csv

### PR DESCRIPTION
**Pull request by skng5**
ICI-2 Set up Quota Report task on Isilon Control

### CODE REVIEW

File: lib/isilon_api/report.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+require 'json'
+require 'csv'
+require 'stringio'
+require 'isilon_api'
+require 'pry'
```
Comment (bibliotechy) [inline lib/isilon_api/report.rb]: Don't need pry, do we?

File: lib/isilon_api/report.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+module IsilonApi
+  module Report
+
+    @@isilon_total_size = 1.0E+15
+    @@default_scale_factor = 1.0E+6
+
+    def self.initialize (csv_filename)
```
Comment (bibliotechy) [inline lib/isilon_api/report.rb]: As per above, can we set the units here instead of in the config file?

File: lib/isilon_api/config.rb [hunk 3]
```diff
@@ -9,14 +9,15 @@ def self.configure()
...
   class Configuration
-    attr_accessor :user, :password, :host, :port
+    attr_accessor :user, :password, :host, :port, :units
 
     def initialize
       @user     = 'example_user'
       @password = '3x4mpl3_p4$$w0rd'
       @host     = 'isilon.example.com'
       @port     = '8080'
+      @units    = 'mb'
```
Comment (bibliotechy) [inline lib/isilon_api/config.rb]: Can we allow the units to be set in the report initialization parameters instead. A global setting for a single report seems unusual here.

**Comment by skng5** [comment]
Units are a command line option.

File: lib/isilon_api/config.rb (hunk 3 above)
Comment (skng5) [inline lib/isilon_api/config.rb]: @bibliotechy It's now a command line option.

File: README.md [hunk 4]
```diff
@@ -27,6 +27,7 @@ IsilonApi.configure do |config|
   config.password = '3x4mpl3_p4$$w0rd'
   config.host     = 'isilon.example.com'
   config.port     = '8080'
+  config.units    = 'mb'
```
Comment (bibliotechy) [inline README.md]: remove units here

File: lib/isilon_api/config.rb [hunk 5]
```diff
@@ -9,7 +9,7 @@ def self.configure()
   end
 
   class Configuration
-    attr_accessor :user, :password, :host, :port
+    attr_accessor :user, :password, :host, :port, :units
```
Comment (bibliotechy) [inline lib/isilon_api/config.rb]: And remove units here as well

**Comment by skng5** [comment]
@bibliotechy : Done. Please verify on actual server.

**Pull request merged**